src/l500/l500-device.h

```diff
@@ -108,6 +112,12 @@ namespace librealsense
 
         std::vector< calibration_change_callback_ptr > _calibration_change_callbacks;
         platform::usb_spec _usb_mode;
+
+        std::mutex _temperature_mutex;
+        std::atomic_bool _temperature_fetcher_on { false };  // Indicates if the temperature thread is working
```

Comment:
Needed? If the thread is `joinable()` then it's on, no?

Reply:
Like discussed this bool indicate the reader thread to keep/stop reading temperatures
I renamed it